### PR TITLE
:dog2: Test for vulnerabilities

### PR DESCRIPTION
**Pull request by st3v3nhunt**
Run synk on `prepush` and i've setup the automated check for PRs

### CODE REVIEW

**Comment by st3v3nhunt** [comment]
Fixes #2

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-redirect-pr-8.dev.c2s.nhschoices.net](http://gp-redirect-pr-8.dev.c2s.nhschoices.net)

File: package.json [hunk 1]
```diff
@@ -41,6 +43,7 @@
     "husky": "^0.13.2",
     "istanbul": "^0.4.5",
     "mocha": "^3.2.0",
-    "nodemon": "^1.11.0"
+    "nodemon": "^1.11.0",
+    "snyk": "https://registry.npmjs.org/snyk/-/snyk-1.25.2.tgz"
```
Comment (neilbmclaughlin) [inline package.json]: Why is this a URL?

Comment (st3v3nhunt) [reply package.json]: Looks like it is a bug https://github.com/npm/npm/issues/15050
I've updated to just the version number and re-generated the shrinkwrap

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-redirect-pr-8.dev.c2s.nhschoices.net](http://gp-redirect-pr-8.dev.c2s.nhschoices.net)

**Review by neilbmclaughlin — APPROVED** [review]

**Pull request merged**